{"uid":"6ef80ad65b8f310c","name":"test_send_message","fullName":"tests.test_example.ExampleTestCase#test_send_message","historyId":"3bc4f3765c9b31fcf733254dd25c049d","time":{"start":1757402276445,"stop":1757402278638,"duration":2193},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_send_message>\n\n    def test_send_message(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe2fea57fd0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1757402270481,"stop":1757402270481,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1757402278661,"stop":1757402278661,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"pkrvm7jw40e0xgp"},{"name":"thread","value":"2634-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c48bce670659c2bf","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55b12f60008a <unknown>\n#1 0x55b12f09fa70 <unknown>\n#2 0x55b12f096cf1 <unknown>\n#3 0x55b12f088099 <unknown>\n#4 0x55b12f089de0 <unknown>\n#5 0x55b12f08843e <unknown>\n#6 0x55b12f087daa <unknown>\n#7 0x55b12f087a85 <unknown>\n#8 0x55b12f0858ee <unknown>\n#9 0x55b12f0860ca <unknown>\n#10 0x55b12f0a2f99 <unknown>\n#11 0x55b12f13dd65 <unknown>\n#12 0x55b12f1173e2 <unknown>\n#13 0x55b12f13d14f <unknown>\n#14 0x55b12f1171b3 <unknown>\n#15 0x55b12f0e359b <unknown>\n#16 0x55b12f0e4971 <unknown>\n#17 0x55b12f5c525b <unknown>\n#18 0x55b12f5c8fa9 <unknown>\n#19 0x55b12f5ac339 <unknown>\n#20 0x55b12f5c9b58 <unknown>\n#21 0x55b12f590c1f <unknown>\n#22 0x55b12f5ed118 <unknown>\n#23 0x55b12f5ed2f6 <unknown>\n#24 0x55b12f5ff066 <unknown>\n#25 0x7fee6109caa4 <unknown>\n#26 0x7fee61129c3c <unknown>","time":{"start":1757402269879,"stop":1757402270656,"duration":777}},{"uid":"fb76b3816ca0fcd5","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402269170,"stop":1757402271419,"duration":2249}},{"uid":"29ffbab69b4f38c","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x5604c36d708a <unknown>\n#1 0x5604c3176a70 <unknown>\n#2 0x5604c316dcf1 <unknown>\n#3 0x5604c315f099 <unknown>\n#4 0x5604c3160de0 <unknown>\n#5 0x5604c315f43e <unknown>\n#6 0x5604c315edaa <unknown>\n#7 0x5604c315ea85 <unknown>\n#8 0x5604c315c8ee <unknown>\n#9 0x5604c315d0ca <unknown>\n#10 0x5604c3179f99 <unknown>\n#11 0x5604c3214d65 <unknown>\n#12 0x5604c31ee3e2 <unknown>\n#13 0x5604c321414f <unknown>\n#14 0x5604c31ee1b3 <unknown>\n#15 0x5604c31ba59b <unknown>\n#16 0x5604c31bb971 <unknown>\n#17 0x5604c369c25b <unknown>\n#18 0x5604c369ffa9 <unknown>\n#19 0x5604c3683339 <unknown>\n#20 0x5604c36a0b58 <unknown>\n#21 0x5604c3667c1f <unknown>\n#22 0x5604c36c4118 <unknown>\n#23 0x5604c36c42f6 <unknown>\n#24 0x5604c36d6066 <unknown>\n#25 0x7fa4f609caa4 <unknown>\n#26 0x7fa4f6129c3c <unknown>","time":{"start":1757402267763,"stop":1757402268428,"duration":665}},{"uid":"65a857b6de4ec068","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402264973,"stop":1757402267564,"duration":2591}},{"uid":"823ec6b8cef6b9f0","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x5580a459308a <unknown>\n#1 0x5580a4032a70 <unknown>\n#2 0x5580a4029cf1 <unknown>\n#3 0x5580a401b099 <unknown>\n#4 0x5580a401cde0 <unknown>\n#5 0x5580a401b43e <unknown>\n#6 0x5580a401adaa <unknown>\n#7 0x5580a401aa85 <unknown>\n#8 0x5580a40188ee <unknown>\n#9 0x5580a40190ca <unknown>\n#10 0x5580a4035f99 <unknown>\n#11 0x5580a40d0d65 <unknown>\n#12 0x5580a40aa3e2 <unknown>\n#13 0x5580a40d014f <unknown>\n#14 0x5580a40aa1b3 <unknown>\n#15 0x5580a407659b <unknown>\n#16 0x5580a4077971 <unknown>\n#17 0x5580a455825b <unknown>\n#18 0x5580a455bfa9 <unknown>\n#19 0x5580a453f339 <unknown>\n#20 0x5580a455cb58 <unknown>\n#21 0x5580a4523c1f <unknown>\n#22 0x5580a4580118 <unknown>\n#23 0x5580a45802f6 <unknown>\n#24 0x5580a4592066 <unknown>\n#25 0x7f84f269caa4 <unknown>\n#26 0x7f84f2729c3c <unknown>","time":{"start":1757402263037,"stop":1757402263715,"duration":678}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6ef80ad65b8f310c.json","parameterValues":[]}